### PR TITLE
docs(project): update installation instructions

### LINKED ISSUE #152: Document build dependencies
In order to get the `libasciidoc` cli binary installed, I had to do the following:

```
    $ go get -u github.com/mna/pigeon
    $ curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
    $ go get -u github.com/bytesparadise/libasciidoc
    $ cd $GOPATH/src/github.com/bytesparadise/libasciidoc
    $ make build
```

Ok to open a quick PR, updating the README with these changes?

### PR DESCRIPTION
**Pull request by moorereason**
Fixes #152

### CODE REVIEW

**Comment by moorereason** [comment]
Replaces #154

File: README.adoc [hunk 1]
```diff
@@ -35,6 +35,8 @@ Further elements will be supported in the future. Feel free to open issues https
 == Installation
 
     $ go get -u github.com/bytesparadise/libasciidoc
+    $ make install-devtools
+    $ make install
```
Comment (xcoulon) [inline README.adoc]: in that case, I would suggest that the `install-devtools` becomes a dependency on the `install` goal, so the user doesn't need to call `make install-devtools` explicitly.

**Comment by xcoulon** [comment]
Thanks for taking care of this issue, @moorereason ! I just suggested a change in the Makefile to make the install process even simpler for the users.

File: README.adoc (hunk 1 above)
Comment (moorereason) [reply README.adoc]: Done

Comment (xcoulon) [reply README.adoc]: awseome! Thanks for this PR, @moorereason!

**Pull request merged**